libs/components/forms/testing/src/checkbox/checkbox-group-harness.ts

```diff
@@ -129,6 +138,18 @@ export class SkyCheckboxGroupHarness extends SkyComponentHarness {
     }
   }
 
+  /**
+   * Whether the checkbox group is stacked.
+   */
+  public async getStacked(): Promise<boolean> {
+    const host = await this.host();
+
+    return (
+      (await host.hasClass('sky-margin-stacked-lg')) ||
```

Comment:
In theory - the consumer could specify these as well, correct?

Should we check for the class and the heading together to ensure that the correct class is there for the heading level and the consumer hasn't just added one of the classes to the host element?

Reply:
like only return `true` if xl is used with a heading or lg is used without a heading? The consumer still could've specified the class and just specified the right one for the scenario. Do you think if they didn't specify it by setting `stacked` we should return false? Or only return true if it is the right combo?

Reply:
> Do you think if they didn't specify it by setting stacked we should return false?

I don’t know that we need to account for accidental correctness. 

> Or only return true if it is the right combo?

Right, I would only return true though if the combination matches correctly.